go/buildhelper/helpertypes/types_test.go

```diff
@@ -0,0 +1,63 @@
+package helpertypes
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
+)
+
+func TestSerialization(t *testing.T) {
+	rol := nodecommon.Rollup{
+		Header: &nodecommon.Header{
```

Comment:
@otherview It seems dangerous to me to use default values in the header here. e.g. if we get back a `withdrawals` of nil after encoding + decoding, is that because the field couldn't be decoded and a default was used, or because we correctly encoded a `nil` value. Same comment for the compression test below.

Reply:
Having the Nonce to 2 should be enough to guarantee that the object is correctly decoded.
I considered being extreme to have one field that is not serialized/decoded correctly.
But I agree that these values should be random and not defaulted, adding a TODO 👍